src/process_viewer.rs

```diff
@@ -338,5 +357,25 @@ fn main() {
         });
     });
 
-    gtk::main();
+    application.add_action(&about);
+    application.add_action(&graphs);
+    application.add_action(&new_task);
+    application.add_action(&quit);
+    application.connect_activate(move |_| {});
+}
+
+fn main() {
+    let application = gtk::Application::new("Process-viewer.myapp", gio::ApplicationFlags::empty())
```

Comment:
Maybe make the application id more unique? com.github.GuillaumeGomez.process-viewer? IIRC most applications also use Java-style "reverse domain names"

Reply:
Oh good idea!